label_studio/core/settings/base.py

```diff
@@ -222,6 +224,11 @@
     'core.current_request.ThreadLocalMiddleware',
 ]
 
+# adds n+1 and eager loading query profiling to logs
+if DEBUG:
```

Comment:
This allows n+1 profile to run when the app is run locally so n+1 queries while running the app, not just during tests.  We might consider putting this behind an additional parameter. But unlike LSE, label studio doesn't have any issues with RQ so this can be run in a normal app run.